server/app/controllers/admin/AdminProgramController.java

```diff
@@ -133,17 +133,16 @@ public Result create(Request request) {
             ImmutableList.copyOf(programData.getCategories()),
             ImmutableList.copyOf(programData.getTiGroups()),
             applicationSteps,
-            programData.getIsCommonIntakeForm()
-                ? ProgramType.COMMON_INTAKE_FORM
-                : ProgramType.DEFAULT);
+            ProgramType.fromValue(programData.getProgramType()));
```

Comment:
(nonblocking) There are a log of calls to ProgramType.fromValue. Consider adding that as a method on programData to simplify the code.